src/NuGet.Core/NuGet.Versioning/VersionRangeFactory.cs

```diff
@@ -9,11 +9,10 @@
 
 namespace NuGet.Versioning
 {
-    /// <summary>
-    /// Static factory methods for creating version range objects.
-    /// </summary>
     public partial class VersionRange
     {
+        // Static factory methods for creating version range objects.
+
```

Comment:
This comment only applies to the members in this partial class's part, but by being a doc comment it is displayed in IntelliSense for the whole class, which seems unintended. Moved to a single line comment within the part to prevent that.